packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1150,6 +1150,11 @@ export interface ThemeIcon {
     color?: ThemeColor;
 }
 
+export namespace ThemeIcon {
+    export function isThemeIcon(arg: any): arg is ThemeIcon {
+        return isObject(arg) && 'id' in arg && 'color' in arg;
+    }
+}
```

Comment:
I don't believe this is necessary? We can import `ThemeIcon` directly in `packages/plugin-ext/src/plugin/scm.ts`

Reply:
Done, and changed.